src/main/resources/db/changelog/changes/schema.xml

```diff
@@ -262,4 +262,102 @@
                                  referencedColumnNames="id"/>
     </changeSet>
 
+    <changeSet id="add_users" author="maxim12321">
+        <insert tableName="user_role">
+            <column name="id" valueSequenceNext="user_role_id_seq"/>
+            <column name="role_name" value="ROLE_USER"/>
+        </insert>
+        <insert tableName="user">
+            <column name="id" valueSequenceNext="user_id_seq"/>
+            <column name="role_id" valueSequenceCurrent="user_role_id_seq"/>
+            <column name="user_login" value="rus_user"/>
```

Comment:
I think we should use emails as the logins.
1. We could use them for email notification
2. Probably we will need to show that it works, so the emails should be real